bitwarden-server.sln

```diff
@@ -76,6 +76,12 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PostgresMigrations", "util\
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Common", "test\Common\Common.csproj", "{17DA09D7-0212-4009-879E-6B9CFDE5FA60}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Infrastructure.Dapper", "src\Infrastructure.Dapper\Infrastructure.Dapper.csproj", "{AD933445-27CE-4D30-A6ED-9065309464AD}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharedWeb", "src\SharedWeb\SharedWeb.csproj", "{713D44C0-1BC1-4024-96A3-A98A49F33908}"
+EndProject
```

Comment:
SharedWeb mostly exists to wire up the DI logic. Since we can't have `Core` depending on the `Infrastructure` projects since it would create a circular dependency. We need a new project to put that logic. I'm not perfectly happy with the naming, or even the existence of the project. But short of going the `Autofac` route which seems even more out of scope, or duplicating the DI hookup (which isn't necessarily a bad idea), I don't see much of an alternative here)